content/guides/accessibility-for-teams/product-management.md

```diff
@@ -0,0 +1,62 @@
+---
+date: 2019-07-31 09:00:00 -0500
+title: "Accessibility for product managers"
+deck: ""
+summary: ""
+guide: accessibility-for-teams
+image: guide-dap
+primary_image: guide-dap
+layout: single
+---
+
+Product managers play a vital role in communicating accessibility requirements early in the project lifecycle, ensuring each team member knows their responsibility, and keeping the team accountable for building accessible products. Following these steps, you’ll make sure you’re not only following legal requirements, but making your product more usable for everyone.
+
+## Why you should care
+
+Any product released by the government must conform to the standards of [Section 508](https://www.section508.gov/) (which aligns with the [W3C Web Content Accessibility Guidelines (WCAG) 2.0 Level AA](https://www.w3.org/WAI/WCAG20/quickref/)), a law that ensures all web content is accessible to anybody with a disability. Not following these requirements poses significant legal risks for any government entity.
+
+If you have project-specific questions, ask your agency’s accessibility team.
+
+## Learn the basics of accessibility and what your team does to make content accessible
+
+### Steps to take
+- Familiarize yourself with the high-level accessibility roles for each member of your team:
+  - [Front end / development]({{< link "/guides/accessibility-for-teams/front-end-development" >}}): Ensure front-end code is written accessibly and conducts manual and automated testing.
+  - [Content design]({{< link "/guides/accessibility-for-teams/content-design" >}}): Ensure content is written in plain language and headings, images, and links are accurately labeled.
+  - [Visual design]({{< link "/guides/accessibility-for-teams/visual-design" >}}): Ensure that designs are accessible, pages are laid out in a logical order, and content meets color contrast requirements.
+  - [UX design]({{< link "/guides/accessibility-for-teams/ux-design" >}}): Ensure that overall experience is built and designed in an accessible fashion, conduct usability testing with people who need accessibility features.
+- Ensure your team members know where to find the accessibility guidelines for their role and follow them from the start of a project. The guides are located here: [front end]({{< link "/guides/accessibility-for-teams/front-end-development" >}}), [content design]({{< link "/guides/accessibility-for-teams/content-design" >}}), and [visual design]({{< link "/guides/accessibility-for-teams/visual-design" >}}).
+- Take an Accessibility 101 class, such as [Udacity’s Web Accessibility class](https://www.udacity.com/course/web-accessibility--ud891), to get a baseline knowledge of accessibility.
+- Learn how to [navigate a webpage using only your keyboard]({{< link "/guides/accessibility-for-teams/visual-design/#keyboard-access" >}}) and [learn how to use VoiceOver]({{< link "/guides/accessibility-for-teams/visual-design" >}}) so you can spot check new features when necessary.
```

Comment:
@bonnieAcameron Used `link` shortcode to reference other guide pages. But, this doesn't allow for setting anchor tags in the link path to navigate to the specific sub-section in the new guide format.

Reply:
Added as a checklist item